libs/sdk-flutter/pubspec.yaml

```diff
@@ -13,6 +13,7 @@ dependencies:
   ### flutter_rust_bridge_codegen dependencies
   ffi: ^2.1.2 # requires Dart >=3.3.0(Flutter 3.19)
   flutter_rust_bridge: ^1.82.6 # breez-sdk requirement
+  flutter_secure_storage: ^9.0.0
```

Comment:
Don't we need to remove the flutter_secure_storage from the sdk?

Reply:
This and the code in libs/sdk-flutter/android/src/main/kotlin/flutter_secured_storage need moving back to c-breez @erdemyerebasmaz